File: DKITProject/DAL/Models/New.cs
using System;

namespace DKITProject.DAL.Models 
{
    public class New 
    {
        public int Id { get; set; }
        public string Headline { get; set; }
        public string Announce { get; set; }
        public string Content { get; set; }
        public string ImgPreview { get; set; }
        public string[] Images { get; set; }
        public DateTime DatePost { get; set; }
        public bool Approved { get; set; }
    }
}

File: DKITProject/ClientApp/src/app/models/new.ts
export class New {
    constructor(
        public id: number,
        public headline: string,
        public announce: string,
        public content: string,
        public imgPreview: string,
        public images: string[],
        public datePost: Date
    ){}
}

export class NewPreview {
    constructor(
        public id: number,
        public headline: string,
        public announce: string,
        public imgPreview: string,
        public datePost: Date
    ){}
}